packages/aws-cdk-lib/aws-cloudtrail/lib/cloudtrail.ts

```diff
@@ -128,6 +128,14 @@ export interface TrailProps {
    */
   readonly isOrganizationTrail?: boolean;
 
+  /** The orgId.
```

Comment:
Why does it call `orgId`? Reaching this [link](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/create-s3-bucket-policy-for-cloudtrail.html), it seems to be `The name of the folder where the log files are stored, including the bucket name, a prefix (if you specified one), and your AWS account ID`. So more like account id?

Reply:
The missing permission is specified under [this section for organization trails](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/create-s3-bucket-policy-for-cloudtrail.html#org-trail-bucket-policy). Specifically [this policy](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/create-s3-bucket-policy-for-cloudtrail.html#:~:text=%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20%22Sid%22%3A%20%22AWSCloudTrailOrganizationWrite20150319,%3Atrail/trailName%22%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%7D) which requires the `o-organizationID`:

```
{
            "Sid": "AWSCloudTrailOrganizationWrite20150319",
            "Effect": "Allow",
            "Principal": {
                "Service": [
                    "cloudtrail.amazonaws.com"
                ]
            },
            "Action": "s3:PutObject",
            "Resource": "arn:aws:s3:::myOrganizationBucket/AWSLogs/o-organizationID/*",
            "Condition": {
                "StringEquals": {
                    "s3:x-amz-acl": "bucket-owner-full-control",
                    "aws:SourceArn": "arn:aws:cloudtrail:region:managementAccountID:trail/trailName"
                }
            }
        }
```

Reply:
This makes a lot more sense! Thanks!